modules/rostests/apitests/win32nt/ntuser/NtUserSetTimer.c

```diff
@@ -7,12 +7,280 @@
 
 #include "../win32nt.h"
 
+//#include <iostream>
+
+#include "windows.h"
+
+#define SLEEP_TIME 500
+#define TIME_TOLERANCE 0.5
+
+#define TEST1_COUNT 20
+#define TEST1_INTERVAL 10
+
+#define TEST2_COUNT 20
+#define TEST2_INTERVAL 10
+
+#define TEST3_COUNT 40000
+#define TEST3_INTERVAL 10
+
+#define TEST4_COUNT 40000
+#define TEST4_INTERVAL 10
+
+typedef struct TIMER_MESSAGE_STATE1 {
+    UINT_PTR index;
+    int counter;
+} TIMER_MESSAGE_STATE1;
+
+TIMER_MESSAGE_STATE1 timerId1[TEST1_COUNT];
+
+typedef struct TIMER_MESSAGE_STATE2 {
+    int counter;
+} TIMER_MESSAGE_STATE2;
+
+TIMER_MESSAGE_STATE2 timerId2[TEST2_COUNT];
+
+void CALLBACK TimerProc(HWND hWnd, UINT uMsg, UINT_PTR idEvent, DWORD dwTime) {
+    for (int i = 0; i < TEST1_COUNT; i++)
+        if (timerId1[i].index == idEvent)
+            timerId1[i].counter++;
+}
+
+void MessageLoop() {
```

Comment:
```suggestion
void MessageLoop(void)
{
```
See https://stackoverflow.com/questions/1163879/in-c-funcvoid-vs-func